src/content/App.tsx

```diff
@@ -10,6 +16,17 @@ const App: React.FC = () => {
     start: 0,
     end: 0,
   });
+  const originalPos = useRef<
+    | {
+        start: number;
+        end: number;
+        currentLine: number;
+      }
+    | undefined
+  >(undefined);
```

Comment:
`|`は見やすくするための対応ですかね？

```suggestion
  const originalPos = useRef<{
        start: number;
        end: number;
        currentLine: number;
      }
    | undefined
  >(undefined);
```